App/Redux/ProcessingImagesRedux.ts

```diff
@@ -3,48 +3,61 @@ import { SharedImage, AddResult } from '../Models/TextileTypes'
 import { RootState } from './Types'
 
 const actions = {
-  insertAddingImage: createAction('processingImages/INSERT_ADDING_IMAGE', resolve => {
-    return (sharedImage: SharedImage, destinationThreadId: string, comment?: string) => resolve({ sharedImage, destinationThreadId, comment })
+  insertImage: createAction('processingImages/INSERT_IMAGE', resolve => {
+    return (uuid: string, sharedImage: SharedImage, destinationThreadId: string, comment?: string) => resolve({ uuid, sharedImage, destinationThreadId, comment })
```

Comment:
Decided to have processing images be tracked with a uuid. This simplified things because we need to track processing images before and after they are added to IPFS, and we don't get a unique id from IPFS until after the image is added to IPFS.